packages/app-store/_utils/paid-apps.ts

```diff
@@ -40,6 +40,8 @@ export const withPaidAppRedirect = async ({
       ? {
           subscription_data: {
             trial_period_days: trialDays,
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore - trial_settings isn't available cc @erik
```

Comment:
I have no idea why this error wasnt caught earlier, but the build is failing now. cc @exception